src/check_release.rs

```diff
@@ -407,7 +419,7 @@ pub(super) fn run_check_release(
             .shell_print(
                 "Checked",
                 format_args!(
-                    "[{:>8.3}s] {} checks; {} passed, {} unnecessary",
+                    "[{:>8.3}s] {} checks; {} passed, {} skipped",
```

Comment:
If we end up making the `passed -> pass / skipped -> skip` change I suggested above, let's make sure we apply the corresponding change here as well for consistency.
```suggestion
                    "[{:>8.3}s] {} checks; {} pass, {} skip",
```